hbw/app/javascript/packs/hbw/components/claiming/created.jsx

```diff
@@ -0,0 +1,91 @@
+/* eslint-disable implicit-arrow-linebreak */
+import React from 'react';
+import {
+  parseISO, isAfter, isSameYear,
+  differenceInMinutes, differenceInHours, differenceInDays, differenceInWeeks, differenceInYears
+} from 'date-fns';
+
+const HBWCreated = ({ env, created, due }) => {
```

Comment:
Extremely bad naming especially if this component can render created or due dates both. It's not even noun!

Reply:
Also I would split this component to two: about due date formatting and about created date formatting, but I don't insist.